### PR TITLE
Fix mobile version for documentation

### LINKED ISSUE #66: Menu show/hide button for mobile version
222891299-20437109-f35b-443e-9db9-6ab3a87a5d15

### PR DESCRIPTION
**Pull request by aggmoulik**
Fixes #66

### CODE REVIEW

**Comment by ije** [comment]
LGTM, thanks

File: components/Header.tsx [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import { Link } from "aleph/react";
+import { Link } from 'aleph/react';
```
Comment (ije) [inline components/Header.tsx:1]: can you please run `deno fmt`? thanks

Comment (aggmoulik) [reply components/Header.tsx]: @ije I have formatted the code, also if any other frontend or docs tasks you want to let me know, I really love working with alephjs and deno, so we can create a great tooling system for deno web environment.

**Review by ije — APPROVED** [review]
LGTM

File: components/Header.tsx (hunk 1 above)
Comment (ije) [reply components/Header.tsx]: thanks! unfortunately currently i don't have too much time on this project...

**Pull request merged**